### DISCUSSION TITLE
randomize quests setting

### DISCUSSION THREAD

**Discussion by ponchikkkk** [diasurgical/DevilutionX#4030] [General]
sorry i am new and a bit confused

Randomize Quests=1
0=disabled, 1=enabled; Adds QoL feature to Disable the Randomized Quests in Single Player. Disables randomly selecting available quests each game, and lets all quests be present in one game. You need to start a new game for the setting to take effect.

should i set this to 0 or 1    to  get all quests ?

**Comment by AJenbo** [comment]
0
